internal/common/types.go

```diff
@@ -250,6 +261,20 @@ func getDistributionMapping() map[string]int {
 	return mapping
 }
 
+func (dist *Distribution) ModulePlatformID() (string, error) {
+	mapping := map[Distribution]string{
+		// TODO: this could be refactored so we don't have these strings hard coded in two places
+		Fedora30: "platform:f30",
+		RHEL82:   "platform:el8",
```

Comment:
f31 and f32 were merged, therefore we should add them here.

Reply:
oh no, more merge conflicts :)